db/internal_stats.h

```diff
@@ -272,10 +272,15 @@ class InternalStats {
     ++cf_stats_count_[type];
   }
 
-  void AddDBStats(InternalDBStatsType type, uint64_t value) {
+  void AddDBStats(InternalDBStatsType type, uint64_t value,
+                  bool concurrent = false) {
     auto& v = db_stats_[type];
-    v.store(v.load(std::memory_order_relaxed) + value,
-            std::memory_order_relaxed);
+    if (concurrent) {
+      v.fetch_add(value, std::memory_order_relaxed);
```

Comment:
hmmm, I did some experiment and fetch_add seems being faster than load+store:
https://gist.github.com/yiwu-arbug/b74461d2a53b1337fbe58406db587a86